tools/PI/DevHome.PI/Views/BarWindowHorizontal.xaml.cs

```diff
@@ -361,17 +365,36 @@ private void Window_Activated(object sender, WindowActivatedEventArgs args)
 
     private void UpdateCustomTitleBarButtonsTextColor()
     {
+        FrameworkElement? rootElement = Content as FrameworkElement;
+        Debug.Assert(rootElement != null, "Expected Content to be a FrameworkElement");
+
+        Windows.UI.Color color;
+        if (rootElement.ActualTheme == ElementTheme.Dark)
+        {
+            color = Colors.White;
+        }
+        else
+        {
+            color = Colors.Black;
+        }
+
         if (_currentActivationState == WindowActivationState.Deactivated)
         {
-            SnapButtonText.Foreground = (SolidColorBrush)Application.Current.Resources["WindowCaptionForegroundDisabled"];
-            ExpandCollapseLayoutButtonText.Foreground = (SolidColorBrush)Application.Current.Resources["WindowCaptionForegroundDisabled"];
-            RotateLayoutButtonText.Foreground = (SolidColorBrush)Application.Current.Resources["WindowCaptionForegroundDisabled"];
+            // Deactivated state has a translucent brush
+            color.A = 0x66;
+            SolidColorBrush brush = new(color);

```

Comment:
If this is constant, perhaps make the brush a field, and create it once?

Reply:
Makes sense to me. Pushed an update.